crates/avalanche-types/src/message/ping.rs

```diff
@@ -41,9 +48,9 @@ impl Message {
         let uncompressed_len = encoded.len();
         let compressed = message::compress::pack_gzip(&encoded)?;
         let msg = p2p::Message {
-            message: Some(p2p::message::Message::CompressedGzip(
-                prost::bytes::Bytes::from(compressed),
-            )),
+            message: Some(p2p::message::Message::CompressedGzip(Bytes::from(
+                compressed,
```

Comment:
FYI, the best times to use `Into` over `From` is when something is being returned or being passed in as an argument.

Reply:
Not part of the change but noted.